lib/battle_boats/board_formatter.rb

```diff
@@ -17,7 +17,7 @@ def format_board(board, hide_ships: true)
           board_string << if cell.occupied? && cell.hit?
                             "  #{cell.occupant.symbol.red}  "
                           elsif cell.occupied? && !hide_ships
-                            "  #{cell.occupant.symbol.yellow}  "
+                            "  #{cell.occupant.symbol.green}  "
```

Comment:
Are we reaching to far into `cell`? (LoD violation?) Is there away to expose this data on an object that closer to us?